apps/account-api/test/accounts.controller.e2e-spec.ts

```diff
@@ -95,4 +97,38 @@ describe('Account Controller', () => {
       .expect((res) => res.body.msaId === validMsaId)
       .expect((res) => res.body.handle.base_handle === handle);
   });
+
+  it('(GET) /v1/accounts/retireMsa/:accountId get payload for retireMsa, given a valid accountId', async () => {
```

Comment:
what should we do about these test?

Reply:
What's the blocker with getting them to work correctly?

Reply:
resolved